modules/apps/foundation/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp

```diff
@@ -234,7 +237,6 @@ name = HtmlUtil.escapeJS(name);
 			plugins.push(
 				{
 					cfg: {
-						uploadItemReturnType: '<%= editorOptions.getUploadItemReturnType() %>',
```

Comment:
Hi @Robert-Frampton !

You are right, sorry! this part of code and the previous one are not related to this issue.